### DISCUSSION TITLE
Generating a Mosaic scenario from SUMO net file

### DISCUSSION THREAD

**Discussion by einho** [eclipse/mosaic#308] [Q&A] [answered]
Hi, I am currently trying to generate a custom mosaic scenario starting from a sumo net file with the scenario-convert tool and the sumo2db command. However, the command line returns an SQL error stating "no such table: main.BuildingCorner". As i was suspicious i even tried to sumo2db the example file but i still get the same error.
Is there a way i can solve this issue? Or is the command is faulty? I'm providing a log of my cmd for further clarification.

15:57:16.517 INFO  - initializing database
15:57:16.520 INFO  - parsing net file
15:57:16.597 INFO  - checking existing routes
15:57:16.597 INFO  - no route file for import given, ignoring!
15:57:16.735 INFO  - Saving 1561 nodes...
15:57:16.747 INFO  - Saving 296 ways...
15:57:16.758 INFO  - Saving 296 connections...
15:57:16.769 INFO  - Saving 26 restrictions...
15:57:16.774 INFO  - Saving 0 routes...
15:57:16.775 INFO  - Saving 0 roundabouts...
15:57:16.775 INFO  - Saving 0 buildings...
15:57:16.775 INFO  - Create Indices
**15:57:16.776 ERROR - error while trying to write database content: [SQLITE_ERROR] SQL error or missing database (no such table: main.BuildingCorner)**
15:57:16.776 INFO  - finished

Thank you in advance,
Andrea

**Comment by schwepmo** [comment, marked as answer]
Hello Andrea,
this is indeed a bug in the script, which is already fixed internally and will be rolled out with the next release (coming soon).
However, if you're not dealing with buildings at all you can safely ignore this error as your scenario database should still be created.
